tests/test_kiwi.py

```diff
@@ -0,0 +1,99 @@
+"""This module contains the tests for the kiwi container, the image with kiwi package & dependencies pre-installed."""
+
+import pytest
+from pytest_container import DerivedContainer
+from pytest_container import container_and_marks_from_pytest_param
+from pytest_container.container import ContainerData
+from pytest_container.container import ImageFormat
+
+from bci_tester.data import KIWI_CONTAINERS
+from bci_tester.runtime_choice import DOCKER_SELECTED
+
+CONTAINER_IMAGES = KIWI_CONTAINERS
+
+KIWI_CONTAINER_EXTENDED = []
+
+CONTAINERFILE_KIWI_EXTENDED = """
+RUN set -euo pipefail; \
+zypper -n in --no-recommends git; \
+zypper -n clean; \
+rm -rf /var/log/{lastlog,tallylog,zypper.log,zypp/history,YaST2}
+
+RUN git clone https://github.com/OSInside/kiwi
+"""
+
+for kiwi_ctr in KIWI_CONTAINERS:
+    ctr, marks = container_and_marks_from_pytest_param(kiwi_ctr)
+    KIWI_CONTAINER_EXTENDED.append(
+        pytest.param(
+            DerivedContainer(
+                base=ctr,
+                containerfile=CONTAINERFILE_KIWI_EXTENDED,
+                image_format=ImageFormat.DOCKER,
+                extra_launch_args=["--privileged", "-v", "/dev:/dev"],
+            ),
+            marks=marks,
+        )
+    )
+
+
+def test_kiwi_installation(auto_container):
+    # check if kiwi package is installed inside the container
```

Comment:
Please make this a docstring rather than a comment, like

```suggestion
   """check if kiwi package is installed inside the container."""
```